### PR TITLE
LibraryPanels: Adds descriptions to cards

### PR DESCRIPTION
**Pull request by hugohaggmark**
**What this PR does / why we need it**:
Adds description to cards with Library Panels.

image

image

image

**Which issue(s) this PR fixes**:
Relates to #33363

**Special notes for your reviewer**:

### CODE REVIEW

**Review by torkelo — COMMENTED** [review]
can we move this into the PanelTypeCard ? think visualizations might want description at some point :)

**Comment by hugohaggmark** [comment]
> can we move this into the PanelTypeCard ? think visualizations might want description at some point :)

Sure, I didn't think that was an option, I mean to extend PanelTypeCard. I'll fix ASAP.

File: public/app/features/library-panels/components/LibraryPanelCard/LibraryPanelCard.tsx [hunk 1]
```diff
@@ -33,38 +33,11 @@ export const LibraryPanelCard: React.FC<LibraryPanelCardProps & { children?: JSX
       <PanelTypeCard
         isCurrent={false}
         title={libraryPanel.name}
+        description={libraryPanel.description}
         plugin={panelPlugin}
         onClick={() => onClick(libraryPanel)}
         onDelete={showSecondaryActions ? () => setShowDeletionModal(true) : undefined}
       />
```
Comment (jackw) [inline public/app/features/library-panels/components/LibraryPanelCard/LibraryPanelCard.tsx:40]: Can we remove the commented out code in this file?

**Review by jackw — APPROVED** [review]
LGTM! 🚀

**Review by dprokop — APPROVED** [review]
@hugohaggmark this looks 👌 

Bumping Jacks comment about leftower commented out code :)

**Comment by jackw** [comment]
> Bumping Jacks comment about leftower commented out code :)
I deleted the comment. Turns out reviewing from a phone is not my strong point. @hugohaggmark had already deleted it. 🙈

**Pull request merged**